djangocms_versioning/datastructures.py

```diff
@@ -11,6 +14,25 @@ def __init__(self, content_model, grouper_field_name):
     def _get_grouper_field(self):
         return self.content_model._meta.get_field(self.grouper_field_name)
 
+    @cached_property
+    def version_proxy(self):
```

Comment:
I don't think this property has been tested. It's a complex enough property that I do think it should have a test.